app/components/global-navigation/search.js

```diff
@@ -411,6 +414,10 @@ export default Component.extend({
 		// ENTER key
 		} else if (keyCode === 13) {
 			if (this.selectedSuggestionIndex !== -1) {
+				if (this.getScope() === 'cross-wiki') {
+					window.location.replace(this.suggestions[this.selectedSuggestionIndex].uri);
```

Comment:
Why `replace` instead on `push`? Is it intentional?

Reply:
We need full reload (redirect to different wiki) and this is just the first solution that worked for me
I can't find `window.location.push` function, so if you can provide snippet i would be grateful :)